package.json

```diff
@@ -17,6 +17,7 @@
     "react-modal": "3.11.2",
     "react-router-dom": "5.1.2",
     "react-scripts": "3.2.0",
+    "react-select": "^3.1.0",
```

Comment:
Let's remove `^` to lock in the library version, and prevent future unexpected library upgrades.

Reply:
Good catch. I have removed it